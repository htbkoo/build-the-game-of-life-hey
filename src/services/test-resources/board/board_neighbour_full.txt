dddldldldddld
dddlldldllldl
dddldldldllld
ldlllldllldld
dldldlldldldl