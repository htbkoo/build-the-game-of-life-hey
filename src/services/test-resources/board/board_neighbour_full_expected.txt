3143645445443
1033544444552
2245745476644
2344646455644
3254745544343